pkgs/servers/openafs/1.8/patches/current_kernel_time.patch

```diff
@@ -0,0 +1,63 @@
+commit 3c454b39d04f4886536267c211171dae30dc0344
```

Comment:
This looks exactly like
http://git.openafs.org/?p=openafs.git;a=patch;h=aa80f892ec39e2984818090a6bb2047430836ee2
So we should be able to use fetchpatch after all.